torch/_dynamo/variables/dicts.py

```diff
@@ -429,6 +430,116 @@ def var_getattr(self, tx, name: str) -> "VariableTracker":
         super().var_getattr(tx, name)
 
 
+class CustomizedDictVariable(ConstDictVariable):
+    @staticmethod
+    @functools.lru_cache(None)
```

Comment:
Using `@functools.lru_cache(None)` here might cause leaks since it will hold a pointer to the user_cls arg.

User classes can get created inside a local scope, so it is possible to create them in a loop.

Reply:
thanks. I removed lru_cache for now. I can improve if there are any better solutions